### PR TITLE
Page Backwards for Release Dynamo Records

### PR DESCRIPTION
**Pull request by nzoschke**

### CODE REVIEW

File: controllers/apps.go [hunk 1]
```diff
@@ -419,6 +419,16 @@ func AppReleases(rw http.ResponseWriter, r *http.Request) {
 	vars := mux.Vars(r)
 	app := vars["app"]
 
+	l := make(map[string]string)
```
Comment (ddollar) [inline controllers/apps.go]: For building this hash you might as well just do

```go
l := map[string]string{
  "id": r.URL.Query().Get("id"),
  "created": created,
}
```

**Comment by ddollar** [comment]
:+1:

**Comment by nzoschke** [comment]
The pagination is working pretty well.

However the build pager interacts poorly with the current live build logs implementation. I will work on that in another pull request.

**Comment by nzoschke** [comment]
Build pager will work as expected once this lands: https://github.com/convox/kernel/pull/56

**Comment by mattmanning** [comment]
:+1:

**Pull request merged**